src/test/groovy/net/fabricmc/loom/BuildUtils.groovy

```diff
@@ -24,10 +24,10 @@ dependencies {
 	//to change the versions see the gradle.properties file
 	minecraft "com.mojang:minecraft:\${project.minecraft_version}"
 	mappings "net.fabricmc:yarn:\${project.yarn_mappings}"
-	modCompile "net.fabricmc:fabric-loader:\${project.loader_version}"
+	compile "net.fabricmc:fabric-loader:\${project.loader_version}"
```

Comment:
:O

Reply:
What?
That is the goal of this whole ordeal :D

Reply:
Well, this should have been `compile` in the example mod, given how the loader touches no minecraft code at all